<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Puces</class>
 <widget class="QDialog" name="Puces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choix de liste</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/programme/images/dadaword.png</normaloff>:/programme/images/dadaword.png</iconset>
  </property>
  <property name="locale">
   <locale language="French" country="Belgium"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Style de listes</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste d√©cimale</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lettres minuscules</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lettres majuscules</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chiffres romains minuscules</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chiffres romains majuscules</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bouton_valide">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/menus/images/ok.png</normaloff>:/menus/images/ok.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_exit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/menus/images/sortir.png</normaloff>:/menus/images/sortir.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
